source/configuration_manual/fts/squat.rst

```diff
@@ -3,11 +3,9 @@
 Squat FTS Engine
 ================
 
-.. deprecated:: v2.1
+.. versionremoved:: v3.0
 
-.. warning:: Squat is deprecated and should no longer be used.
-             It is going to be removed in a future version entirely.
-	     It's recommended to use :ref:`fts_backend_solr` instead.
+.. warning:: It's recommended to use :ref:`fts_backend_solr` instead.
```

Comment:
The sentence no longer sounds like a warning. Maybe it's a "note" now?

Reply:
here too, the previous cases retained a warning (autocreate-plugin, expire-plugin, both in 2.3.14)
want me to turn them all into notes ?